syntax = "proto3";
import "base.proto";
import "user.proto";

service Booking {
    rpc GetAllBookings(Empty) returns (stream BookingData) {}
    rpc GetBookingByUserId(UserID) returns (BookingData) {}
}

message BookingData {
    string userId = 1;
    repeated DateItem date = 2;
}

message DateItem {
    string date = 1;
    repeated string moviesId = 2;
}